packages/grafana-e2e/src/typings/undo.ts

```diff
@@ -0,0 +1,18 @@
+const platforms = {
+  osx: 'darwin',
+  windows: 'win32',
+  linux: 'linux',
+  aix: 'aix',
+  freebsd: 'freebsd',
+  openbsd: 'openbsd',
+  sunos: 'sunos',
+};
+
+export const undo = () => {
+  switch (Cypress.platform) {
+    case platforms.osx:
```

Comment:
Nit: An enum might be more appropriate

```suggestion
enum Platform {
  osx = 'darwin',
  windows = 'win32',
  linux = 'linux',
  aix = 'aix',
  freebsd = 'freebsd',
  openbsd = 'openbsd',
  sunos = 'sunos',
}

export const undo = () => {
  switch (Cypress.platform) {
    case Platform.osx:
```

Reply:
Also might leave a comment indicating where the values for `platform` come from (https://nodejs.org/api/os.html#os_os_platform)

Reply:
Great feedback! I will update, thanks!